src/GovUk.Education.ExploreEducationStatistics.Common/ViewModels/ValidationProblemViewModel.cs

```diff
@@ -0,0 +1,60 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+
+public class ValidationProblemViewModel : ValidationProblemDetails
+{
+    public new int Status = StatusCodes.Status400BadRequest;
+
+    /// <summary>
+    /// Additional errors that are related to the problem.
+    /// </summary>
+    public new IReadOnlyList<ErrorViewModel> Errors { get; set; } = new List<ErrorViewModel>();
+
+    [JsonIgnore]
+    [Newtonsoft.Json.JsonIgnore]
+    public ValidationProblemDetails? OriginalDetails { get; init; }
+
+    public static ValidationProblemViewModel Create(ValidationProblemDetails problemDetails)
+    {
+        var errors = problemDetails.Errors
+            .SelectMany(errorEntry =>
+                errorEntry.Value.Select(
+                    message => new ErrorViewModel
+                    {
+                        Path = errorEntry.Key.IsNullOrEmpty() ? null : errorEntry.Key,
+                        Message = message
+                    }
+                )
+            )
+            .ToList();
+
+        return Create(problemDetails, errors);
+    }
+
+    public static ValidationProblemViewModel Create(
+        ValidationProblemDetails problemDetails,
+        IEnumerable<ErrorViewModel> errors)
+    {
+        var viewModel = new ValidationProblemViewModel
+        {
+            Title = problemDetails.Title,
+            Type = problemDetails.Type,
+            Status = problemDetails.Status ?? default,
```

Comment:
I think this will end up defaulting the `Status` to `0` if `problemDetails.Status` is `null`. I'd just remove the line completely and let the default `400` status you've set with the `new` keyword take over